### PR TITLE
New package: yersinia-v0.8.2

### PR DESCRIPTION
**Pull request by misuchiru03**

### CODE REVIEW

**Comment by abenson** [comment]
Commit message should be "New package: ${pkgname}-${version}"

**Comment by abenson** [comment]
[Looking good to me](https://www.youtube.com/watch?v=B0azMOJ-h_o)

**Comment by abenson** [comment]
@maxice8 please review

File: srcpkgs/yersinia/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+hostmakedepends="pkg-config automake"
+makedepends="libpcap-devel libnet-devel ncurses-devel gettext-devel"
+short_desc="Framework for layer 2 security testing"
+maintainer="Alex Childs <misuchiru03+void@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/tomac/yersinia"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=964f528dbffd904376ab7986e69b1e8639591c929e2f39186ac9ebf2d3bc761f
+
+if [ -n "$CROSS_BUILD" ]; then
```
Comment (maxice8) [inline srcpkgs/yersinia/template]: no need for -n

**Pull request merged**